osi_sensorviewconfiguration.proto

```diff
@@ -223,6 +223,11 @@ message SensorViewConfiguration
     //
     repeated UltrasonicSensorViewConfiguration
         ultrasonic_sensor_view_configuration = 1004;
+        
+    // Raytracer-specific View Configuration(s).
+    //
+    repeated RaytracerViewConfiguration
```

Comment:
As written in the comment in `osi_sensorview.proto`, we should clarify whether ray tracing should be written separately or not.

Reply:
Changed to RayTracerViewConfiguration